metron-analytics/metron-profiler/src/main/assembly/assembly.xml

```diff
@@ -63,7 +63,7 @@
         <fileSet>
             <directory>${project.basedir}/target</directory>
             <includes>
-                <include>${project.artifactId}-${project.version}.jar</include>
+                <include>${project.artifactId}-${project.version}-uber.jar</include>
             </includes>
```

Comment:
I'd be glad to check.  If you can point me to a README or specifics on what to check that would be appreciated.